ra2ce/analysis/indirect/losses.py

```diff
@@ -52,28 +52,22 @@ def __init__(self, config: AnalysisConfigData, analysis: AnalysisSectionIndirect
         self.network = self._load_gdf(
             self.losses_input_path.joinpath("network.geojson")
         )
-        self.intensities = self._load_df_from_csv(
-            self.losses_input_path.joinpath("traffic_intensities.csv"), []
-        )  # per day
+        self.intensities = self._load_df_from_csv(analysis.traffic_intensities_file, [])  # per day
 
         # TODO: make sure the "link_id" is kept in the result of the criticality analysis
-        self.criticality_data = self._load_df_from_csv(
-            self.losses_input_path.joinpath("criticality_data.csv"), []
-        )
-        self.resilience_curve = self._load_df_from_csv(
-            self.losses_input_path.joinpath(analysis.resilience_curve_file),
-            ["disruption_steps", "functionality_loss_ratio"],
-        )
-        self.values_of_time = self._load_df_from_csv(
-            self.losses_input_path.joinpath("values_of_time.csv"), []
-        )
+        self.criticality_data = self._load_df_from_csv(Path("criticality_data.csv"), [])
```

Comment:
this is is not correct, when you do `Path("something")` it becomes relative to the execution context, so you cannot know with exactitude where the file is located, please either revert it to the previous value or set it as another property (yet I still believe everything should be relative to the losses input directory).

Reply:
discussed with @MatthiasHauthDeltares , this will be fixed later on a different branch.